### PR TITLE
Fix no color in console log output on windows

### PR DESCRIPTION
**Pull request by linhaojun857**

### CODE REVIEW

File: src/oatpp/core/base/Environment.cpp [hunk 1]
```diff
@@ -76,24 +106,19 @@ void DefaultLogger::log(v_uint32 priority, const std::string& tag, const std::st
...
-      break;
+#ifdef _WIN32
+      SetConsoleColor(LOG_CONST_TABLE[priority][0]);
+      std::cout << " " << (char)LOG_CONST_TABLE[priority][1] << " ";
+      SetConsoleColor(FOREGROUND_DEFAULT);
+      std::cout << "|";
+#else
+      std::cout << LOG_CONST_TABLE[priority][0];
+      std::cout << " " << LOG_CONST_TABLE[priority][1] << " " << FOREGROUND_DEFAULT << "|";
+#endif
```
Comment (lganzzzo) [inline src/oatpp/core/base/Environment.cpp:121]: Looks like there is no `break` statement. 
Also, we can use `if/else` here instead of `switch`

**Review by lganzzzo — COMMENTED** [review]
Hello @linhaojun857 ,

Thanks for the PR!
Please fix minor comments.

Regards,
Leonid

File: src/oatpp/core/base/Environment.cpp (hunk 1 above)
Comment (linhaojun857) [reply src/oatpp/core/base/Environment.cpp:116]: Hello, my pr has been open for a long time, can it be merged?

**Comment by linhaojun857** [comment]
Hello, my pr has been open for a long time, can it be merged?

**Pull request closed without merging**